docs/getting-started-guides/kubeadm.md

```diff
@@ -125,7 +133,33 @@ If you want to be able to schedule pods on the master, for example if you want a
 
 This will remove the "dedicated" taint from any nodes that have it, including the master node, meaning that the scheduler will then be able to schedule pods everywhere.
 
-### (3/4) Joining your nodes
+### (3/4) Installing a pod network
+
+You must install a pod network add-on so that your pods can communicate with each other.
+**It is necessary to do this before you try to deploy any applications to your cluster, and before `kube-dns` will start up.**
+
+Several projects provide Kubernetes pod networks.
+You can see a complete list of available network add-ons on the [add-ons page](/docs/admin/addons/).
+
+By way of example, you can install [Weave Net](https://github.com/weaveworks/weave-kube) by logging in to the master and running:
+
+    # kubectl apply -f https://git.io/weave-kube
+    daemonset "weave-net" created
+
+If you want to use flannel, run this:
+
+    # ARCH=amd64 curl -sSL https://raw.githubusercontent.com/luxas/flannel/update-daemonset/Documentation/kube-flannel.yml | sed "s/amd64/${ARCH}/g" | kubectl create -f -
```

Comment:
I understand what you are trying to do, but it's a bit confusing as it stand - sed expression has no effect...